### PR TITLE
[docs] Development Builds: use `kbd` element to represent keyboard shortcuts

### PR DESCRIPTION
**Pull request by amandeepmittal**
# Why

Refs ENG-5555

# How

As per our new [style guide](https://www.notion.so/expo/Getting-Started-Tutorial-outline-76189e5977d34d9b976aa6f00757a023), we should use `<kbd>` elements in markdown files to represent keyboard shortcuts.

# Test Plan

Changes tested by running docs locally.

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [ ] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Review by Simek — COMMENTED** [review]
We need to add a design for those element first, since currently those tags are currently unstyled.

I have asked Jon to prepare mocks for this, when those are ready, new appearance need to be coded.

**Comment by amandeepmittal** [comment]
@Simek, let me know if I can help add those new components.

For this PR, I can leave it open or remove the <kbd> part. I think we can circle this one back when we have those components.

**Comment by Simek** [comment]
Sure, I will prepare PR soon. One question, would like to use monospace font for the keys or not? By default in browsers monospace font is used, but referenced blog post use serif font.

**Comment by jonsamp** [comment]
> Sure, I will prepare PR soon. One question, would like to use monospace font for the keys or not? By default in browsers monospace font is used, but referenced blog post use serif font.

Serif makes sense to me. Most keyboards do not actually have monospace fonts, and it will make these elements look different than inline code.

File: docs/pages/development/getting-started.md [hunk 1]
```diff
@@ -181,4 +181,4 @@ const styles = StyleSheet.create({
 
 ## Debugging your app
 
-When you need to, you can access the menu by pressing Cmd-d in Expo CLI or by shaking your phone or tablet. Here you'll be able to access all of the functions of your development build, access any debugging functionality you need, or switch to a different version of your app.
+When you need to, you can access the menu by pressing <kbd>⌘</kbd>+<kbd>d</kbd>/<kbd>ctrl</kbd>+<kbd>d</kbd> in Expo CLI or by shaking your phone or tablet. Here you'll be able to access all of the functions of your development build, access any debugging functionality you need, or switch to a different version of your app.
```
Comment (Simek) [inline docs/pages/development/getting-started.md:184]:
```suggestion
When you need to, you can access the menu by pressing <kbd>⌘</kbd> + <kbd>D</kbd> or <kbd>Ctrl</kbd> + <kbd>D</kbd> in Expo CLI or by shaking your phone or tablet. Here you'll be able to access all of the functions of your development build, access any debugging functionality you need, or switch to a different version of your app.
```

Just formatting nits. From my tests while working on the styling, shortcuts looks the best if we add spaces between pluses.

File: docs/pages/development/getting-started.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ Instead, all you need to do to start developing is to run:
 
 and scanning the resulting QR code with your system camera or QR code reader (if you want to develop against a physical device)
 
-or pressing the "a" or "i" keys (to open the app in your Android or iPhone emulator or simulator respectively).
+or pressing the <kbd>a</kbd> or <kbd>i</kbd> keys (to open the app in your Android or iPhone emulator or simulator respectively).
```
Comment (Simek) [inline docs/pages/development/getting-started.md:72]:
```suggestion
or pressing the <kbd>A</kbd> or <kbd>I</kbd> keys (to open the app in your Android or iPhone emulator or simulator respectively).
```

Capitalize keys.

**Review by Simek — APPROVED** [review]
Tested the changes locally and it looks good, thank you for the update! 🙏

**Pull request merged**